cli/config-cli/src/test/java/com/quorum/tessera/config/cli/PicoCliDelegateTest.java

```diff
@@ -608,4 +608,18 @@ public void withValidConfigAndJdbcOverides() throws Exception {
         assertThat(config.getJdbcConfig().isAutoCreateTables()).isTrue();
         assertThat(config.getJdbcConfig().getUrl()).isEqualTo("someurl");
     }
+
+    @Test
+    public void withRecoverMode() throws Exception {
+        Path configFile = createAndPopulatePaths(getClass().getResource("/sample-config.json"));
+        CliResult result = cliDelegate.execute("-configfile", configFile.toString(), "-r");
+
+        assertThat(result).isNotNull();
+        assertThat(result.getConfig()).isPresent();
```

Comment:
This line is duplicated - and looks like it was duplicated by copying, in the commit that added the same stuff 4 times in this file. Is the duplication deliberate?

Reply:
Thanks for pointing out - it's copying error - now resolved